pkg/ts/core/Arc.ts

```diff
@@ -178,7 +178,7 @@ export class Arc extends EventEmitter {
     if (this.composer) {
       this.composer.render(packet);
     } else {
-      this.log('render called, but composer is null', packet);
+      //this.log.low('render called, but composer is null', packet);
```

Comment:
Intentional?

Reply:
Yes, this message comes up too often, so I disabled it, but did not remove it so we could consider inventing a 'low' (priority) method for 'verbose' logging.